packages/website/docs/components/patterns/AI Writing Assistant/AI Writing Assistant.mdx

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_class_name: newComponentBadge expComponentBadge
+---
+
+import Input from "../../../components/patterns/AI Writing Assistant/Input.md";
+
+## Overview
+The AI Writing Assistant streamlines interactions with generative AI, enhancing task
+completion efficiency and making the experience intuitive and valuable for users.
+
+## AI writing assistant for Input fields
+
+It appears within input field component to assist users in creating, iterating,
+and improving text entries through Quick Prompts. These instructions for generative
+AI are crafted by experts known as prompt engineers, who focus on prompt quality.
+The AI Writing Assistant Menu provides writing-specific prompts such as “Change Tone,”
+Adjust Length,” Translate,” and “Analyze Text,” which users can apply to the entire text
+or a selected portion.
```

Comment:
The AI Writing Assistant appears within input field components to help users create, iterate, and improve text entries using Quick Prompts. Experts, known as prompt engineers, craft these generative AI instructions with a focus on prompt quality. The AI Writing Assistant Menu offers writing-specific prompts such as "Change Tone," "Adjust Length," "Translate," and "Analyze Text." Users can apply these prompts to the entire text or just a selected portion.